tools/build_changelog

```diff
@@ -59,7 +59,11 @@ function build_section
 	directory=$1
 	lastbranch=$2
 	cd $directory || fail "Unable to cd to $directory"
-	description=${3:-$(basename $PWD)}
+	if [[ "$directory" = "." ]]; then
```

Comment:
This is an obscure enough corner that it merits a comment IMHO

Reply:
Yep, will do.